sandbox/src/mocks/stu3/cancelReferral/responses/CancelledBookedReferralReferrerCancellation.json

```diff
@@ -112,6 +120,14 @@
                                     "code": "HEART_FAILURE"
                                 }
                             },
+                            {
+                                "valueString": "LS1 2TW",
+                                "name": "postcode"
```

Comment:
These two lines seem to be the wrong way round.

Reply:
The order doesn't actually matter, so long as both 'valueString' and 'name' fields are present.